### PR TITLE
Add AdminKit

### PR DESCRIPTION
**Pull request by paullaros**

### CODE REVIEW

**Comment by XhmikosR** [comment]
Please run the screenshots through https://kraken.io/web-interface (lossless) or lossy if it still looks good.

**Comment by XhmikosR** [comment]
Also @jdorfman is there any discussion I'm missing about this?

**Comment by XhmikosR** [comment]
The images are bigger now :P

**Comment by XhmikosR** [comment]
@paullaros I squashed your patches, so make sure you don't overwrite the changes.

I wait for @jdorfman to chime in.

**Comment by paullaros** [comment]
@XhmikosR Thanks!

File: config/_extras.yml [hunk 1]
```diff
@@ -18,6 +18,12 @@ themesAd:
   - name: 'Pixel Lite - Bootstrap 4 UI Kit'
     img: /assets/img/themes/pixel-lite-free-ui-kit-themesberg.jpg
     url: 'https://themesberg.com/product/ui-kits/pixel-lite-free-bootstrap-4-ui-kit'
+  - name: 'AdminKit Pro - Web UI Kit & Dashboard Template'
+    img: /assets/img/themes/adminkit-pro-ui-kit-dashboard.jpg
+    url: 'https://adminkit.io/'
+  - name: 'AdminKit - Web UI Kit & Dashboard Template'
```
Comment (jdorfman) [inline config/_extras.yml]: This one can be added

File: config/_extras.yml [hunk 2]
```diff
@@ -18,6 +18,12 @@ themesAd:
   - name: 'Pixel Lite - Bootstrap 4 UI Kit'
     img: /assets/img/themes/pixel-lite-free-ui-kit-themesberg.jpg
     url: 'https://themesberg.com/product/ui-kits/pixel-lite-free-bootstrap-4-ui-kit'
+  - name: 'AdminKit Pro - Web UI Kit & Dashboard Template'
```
Comment (jdorfman) [inline config/_extras.yml]: This can't, other affiliates will ask why they have to give us a cut of each sale.

**Comment by paullaros** [comment]
Sure! Thanks 👍

**Comment by XhmikosR** [comment]
@paullaros please squash all the patches :)

**Comment by jdorfman** [comment]
@XhmikosR merge whenever thanks @paullaros

**Pull request merged**